SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -175,6 +177,7 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	void InitiateConnectionToSpatialOS(const FURL& URL);
 
 	void InitializeSpatialOutputDevice();
+
```

Comment:
I think it's okay if there isn't a newline since these two functions are supposed to be grouped together.